save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -734,30 +741,35 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         }
     }
 
-    private fun postFileDelete(file: FileInfo) {
+    private fun fileDelete() {
         scope.launch {
-            val headers = Headers().also {
-                it.set("Accept", "application/json")
-                it.set("Content-Type", "application/json")
-            }
-
             val response = delete(
-                "$apiUrl/files/${props.owner}/${props.name}/${file.uploadedMillis}",
-                headers,
-                Json.encodeToString(file),
+                "$apiUrl/files/${props.owner}/${props.name}/${state.file?.uploadedMillis}",
```

Comment:
Does it make sense to have URL like `/saveourtool/diktat/null` here? I think you can save `state.file` to a local varialbe with nullability check, because if `state.file == null` this request won't make sense